public_html/composer.json

```diff
@@ -0,0 +1,129 @@
+{
```

Comment:
`SHOULD` - If there's a practical way to install this in the root of the project, that'd be ideal. If we avoid WPackagist like suggested below, that might be easier.

Reply:
At first glance, this sounds like it'd work: https://github.com/composer/installers#custom-install-paths

Reply:
More: https://github.com/outlandishideas/wpackagist/issues/1

Reply:
Thanks for the leads. They helped me figure it out, so it's back in the root with a3d925eb

Sidenote, I also researched whether the **composer.json** file could live in a directory like **.config** or something, instead of the root. It appears that it's only possible by setting an environment variable, which would then apply everywhere you use composer locally, and could make things really confusing, in addition to it being an extra, unnecessary burden on contributors. So it's staying in the root for now.

Reply:
👍